src/docs/sphinx/dev_guide/component_org.rst

```diff
@@ -138,15 +138,15 @@ files or those in other components for more details.
 Top-level component directory
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The top-level component directory contains a `CMakeLists.txt`, e.g., 
-`axom/src/components/sidre/CmakeLists.txt`, which contains the following items:
+The top-level component directory contains a ``CMakeLists.txt``, e.g., 
+``axom/src/components/sidre/CMakeLists.txt``, which contains the following items:
```

Comment:
Maybe add a note here that the dependency info for each component must be changed when its dependencies change?

Reply:
Clarified that dependencies should only be listed in a component when they are directly used in that component. Otherwise ensure that they are in the upstream component.  Also added a comment about optional dependencies.